app/views/user/check_email.html.slim

```diff
@@ -1,20 +1,23 @@
 .govuk-grid-row
   .govuk-grid-column-two-thirds-from-desktop
     h1.govuk-heading-l Check your email
-
-    p.govuk-body
-      | We have sent you an email with instructions for how to verify your email address.  We sent the email to:
-
-    <p class="govuk-body govuk-!-font-weight-bold">
-      = @user.email
-    </p>
-
-    p.govuk-body
-      | To continue setting up your training account you must click the link in the email. If you have not received the email after a few minutes, please check you spam folder.
+      
+    - if @user
+      p.govuk-body
+        | We have sent you an email with instructions for how to verify your email address. We sent the email to:
+      p class="govuk-body govuk-!-font-weight-bold"
+        = @user.email
+      p.govuk-body
+        | To continue setting up your training account you must click the link in the email. If you have not received the email after a few minutes, please check your spam folder.
+    - else 
+      p.govuk-body 
+        | We have sent you an email with instructions for how to reset your password.
+      p.govuk-body
+        | If you have not received the email after a few minutes, please check your spam folder.
 
 .govuk-grid-row
   .govuk-grid-column-two-thirds-from-desktop
     = govuk_details(summary_text: "Need us to resend the email?") do
-      = govuk_link_to "I haven't received the email", new_confirmation_path(@user)
+      = govuk_link_to "I haven't received the email", @user ? new_confirmation_path(@user) : new_password_path(:user)
```

Comment:
I'd like to see this path as a variable set in the controller would there be any objections?

Reply:
this is why I would use two templates, as above.  there are only passing similarities between the two pages, with those possibly extractable into translation calls....